assets/questions/findChar.js

```diff
@@ -0,0 +1,114 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['findChar'] = { // eslint-disable-line dot-notation
+  title: 'Find the first character in a string that doesn\'t repeat',
+  starterCode: {
+    python: `def find1stNonRepeatedChar(word):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python: `class AuxiliaryCode(object):
+    @classmethod
+    def sortChar(cls, word):
+        sortedStr = ''.join(sorted(word))        
+        c = [a for a in sortedStr if sortedStr.count(a) == 1]
+        if not c:
+            return None
+        else:
+            return c[0]
+`
+  },
+  tasks: [
+    {
+      instructions: [
+        [
+          'For this question, you\'ll implement the find1stNonRepeatedChar function. ',
+          'Write a function to find the first character in a String that doesn\'t repeat.'
+        ].join(''),
+        'For instance, \'mom\' would output \'o\'.'
+      ],
+      prerequisiteSkills: ['Strings', 'String Manipulation'],
+      acquiredSkills: ['String Manipulation'],
+      inputFunctionName: null,
+      outputFunctionName: null,
+      mainFunctionName: 'find1stNonRepeatedChar',
+      correctnessTests: [
+        {input: 'mom', 
+          allowedOutputs: ['o']},
+        {input: 'apple', 
+          allowedOutputs: ['a']},
+        {input: 'wefffa', 
+          allowedOutputs: ['w']},
+        {input: 'weffwa', 
+          allowedOutputs: ['e']},
+        {input: '', 
+          allowedOutputs: [null]},
+        {input: 'aaaeee', 
+          allowedOutputs: [null]}
+      ],
+      buggyOutputTests: [{
+        buggyFunctionName: 'AuxiliaryCode.sortChar',
+        messages: [[
+          'Try running your code on \'wefffa\' on paper. What\'s the result?   ',
+          'Are you sure that you\'re returning the first non-repeating character in the word?   ',
+          'It looks like you\'re returning the character that is unique but not the first one in the given string.'
+        ].join('')]
+      }],
+      performanceTests: []
+    },
+    {
+      instructions: [[
+        'Next, double-check your code to make sure that it handles ',
+        'more than just letters.'
+      ].join('')],
+      prerequisiteSkills: ['Arrays', 'Strings', 'String Manipulation'],
+      acquiredSkills: ['String Manipulation', 'Sets', 'Arrays', 'Maps'],
+      inputFunctionName: null,
+      outputFunctionName: null,
+      mainFunctionName: 'find1stNonRepeatedChar',
+      correctnessTests: [
+        {input: 'TT AsAs', 
+          allowedOutputs: [' ']},
+        {input: 'AbCABcBb', 
+          allowedOutputs: ['C']},
+        {input: 'BB337 7122 ', 
+          allowedOutputs: ['1']},
+        {input: 'test11@test', 
+          allowedOutputs: ['@']}
+      ],
+      buggyOutputTests: [],
+      performanceTests: [{
```

Comment:
Since we are only testing performance, so it doesn't matter if there exists a unique character or not. Considering common used solutions to this question, a string without any unique characters is the worst case, in other words it requires longest time to solve, and would make differences between solutions of different time complexity clearer.

About space, I just took it as a special character in a string, like '@' character I used in another test case. But I agree with you that there shouldn't be spaces by intuition.